src/Infrastructure/Service/ActionApprover.php

```diff
@@ -9,18 +9,19 @@
 
 use SprykerSdk\Sdk\Core\Application\Dependency\ActionApproverInterface;
 use SprykerSdk\Sdk\Core\Application\Dto\ReceiverValue;
+use SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver;
 
 class ActionApprover implements ActionApproverInterface
 {
     /**
-     * @var \SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver
      */
-    protected CliValueReceiver $cliValueReceiver;
+    protected CliReceiver $cliValueReceiver;
```

Comment:
It must be depended on the abstraction. Please change it to the `ReceiverInterface` interface.